### PR TITLE
Don't use work type for edit button, ref #377

### LINKED ISSUE #377: Use title for work when editing
The work type id is being displayed in the page and update button. This should be changed to the title of the work.

Remove the title from the button, and ensure title is displayed at the top of the page. Button should say "Save Work" using i18n.

### PR DESCRIPTION
**Pull request by awead**
Fixes #377

## Description

The work type's id was inadvertently being used as the edit button. This
changes the button to a i18n value for easier configuration. It also
includes i18n values for delete buttons, as well as create views.

## Changes

Are there any major changes in this PR that will change the release process? No.

## Checklist

- [x] i18n enabled
- [x] adequate test coverage
- [x] accessible interface

### CODE REVIEW

File: app/views/work/submissions/edit.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<h1>Editing <%= @work.work_type.titleize %></h1>
+<h1><%= t('cho.work.edit.heading') %></h1>
```
Comment (cam156) [inline app/views/work/submissions/edit.html.erb]: @awead I would expect this to have the title of the work you are editing

File: app/views/work/submissions/new.html.erb [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <% work_type = Work::Type.find(Valkyrie::ID.new(@work.work_type)) %>
 
 <% if work_type %>
-  <h1>New <%= work_type.label %> Submission</h1>
+  <h1><%= t('cho.work.new.heading', type: work_type.label) %></h1>
```
Comment (cam156) [inline app/views/work/submissions/new.html.erb]: 👍  This one looks right!

File: app/views/work/submissions/edit.html.erb (hunk 1 above)
Comment (awead) [reply app/views/work/submissions/edit.html.erb]: @cam156 we talked about this yesterday and titles can be quite long. Let me verify with @ntallman

**Comment by cam156** [comment]
@awead feel free to rebase and merge.  Not sure why the rebase is necessary.

File: app/views/work/submissions/edit.html.erb (hunk 1 above)
Comment (awead) [reply app/views/work/submissions/edit.html.erb]: @cam156 the title is in an h2 tag now.

**Pull request merged**